Makefile

```diff
@@ -0,0 +1,55 @@
+SHELL=bash
+
+.PHONY: help
+help:
+	@echo "Please specify one of the following:"
+	@echo ""
+	@echo "deps: install the dependencies of the project for your platform"
+	@echo "install: install the application"
+	@echo "certs: builds and installs the certificates"
+	@echo "images: installs basic simphony images"
+	@echo "db: initializes the sqlite database"
+	@echo ""
+
+deps:
+	@echo "Installing dependencies"
+	@echo "-----------------------"
+	sudo apt-get update
+	sudo apt-get install docker-engine npm nodejs-legacy python3-pip python3.4-venv
+	sudo npm install -g configurable-http-proxy
+
+.PHONY: install
+install: 
+	@echo "Installing application"
+	@echo "----------------------"
+	python3 -mvenv venv
+	. venv/bin/activate \
+		&& pip3 install docker-py \
+		&& python3 setup.py develop 
```

Comment:
Instead of doing a `pip3 install` for the `install` target, I'd prefer the make target be called `develop` for `setup.py develop`.  Maybe that's easier for now.